<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Viewer</class>
 <widget class="QWidget" name="Viewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="layout_Invers_and_Setting_button" stretch="0,1,1">
     <item>
      <widget class="QCheckBox" name="inversion">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>negative of  image</string>
       </property>
       <property name="text">
        <string>Inversion</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="marker_label">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_settings">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout_graphicView_and_Pix_filter_panel">
     <item>
      <widget class="QGraphicsView" name="graphicsView">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="backgroundBrush">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>200</red>
          <green>200</green>
          <blue>200</blue>
         </color>
        </brush>
       </property>
       <property name="foregroundBrush">
        <brush brushstyle="NoBrush">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </property>
       <property name="interactive">
        <bool>true</bool>
       </property>
       <property name="sceneRect">
        <rectf>
         <x>0.000000000000000</x>
         <y>0.000000000000000</y>
         <width>0.000000000000000</width>
         <height>0.000000000000000</height>
        </rectf>
       </property>
       <property name="renderHints">
        <set>QPainter::Antialiasing|QPainter::HighQualityAntialiasing|QPainter::SmoothPixmapTransform|QPainter::TextAntialiasing</set>
       </property>
       <property name="cacheMode">
        <set>QGraphicsView::CacheNone</set>
       </property>
       <property name="transformationAnchor">
        <enum>QGraphicsView::AnchorUnderMouse</enum>
       </property>
       <property name="resizeAnchor">
        <enum>QGraphicsView::AnchorUnderMouse</enum>
       </property>
       <property name="viewportUpdateMode">
        <enum>QGraphicsView::FullViewportUpdate</enum>
       </property>
       <property name="rubberBandSelectionMode">
        <enum>Qt::IntersectsItemShape</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
